Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -703,6 +712,9 @@ private static void RegisterInstance(MixedRealityToolkit toolkitInstance)
 
         private static void UnregisterInstance(MixedRealityToolkit toolkitInstance)
         {
+            // We are shutting this instance down.
+            internalShutdown = true;
```

Comment:
>internalShutdown = true; [](start = 12, length = 24)

Should RegisterInstance set this back to true? Otherwise it seems like even the static global will always be shutdown forever...

Reply:
good catch!